src/static/manifest.json

```diff
@@ -1,11 +1,11 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "__MSG_manifest_name__",
   "short_name": "__MSG_manifest_short_name__",
   "description": "__MSG_manifest_description__",
-  "version": "2024.9.11",
+  "version": "2024.9.23",
```

Comment:
Could you do a global search-and-replace of 2024.9.11 to 2024.9.23? Tests will fail otherwise.

Reply:
I thought it was just one place. THis should now be updated.